architecture/2024-08-dashboard-component-threat-model.md

```diff
@@ -0,0 +1,299 @@
+# Radius Dashboard Component Threat Model
+
+- **Author**: @nithyatsu
+
+## Overview
+
+This document provides a threat model for the Radius Dashboard component. It identifies potential security threats to this part of Radius and suggests possible mitigations. The document includes an analysis of the system, its assets, identified threats, and recommended security measures to protect the system.
+
+The Radius Dashboard component provides the frontend experience for Radius. 
+It provides visual and textual representation of user's applications, environments and recipes.
+
+## Terms and Definitions
+
+| Term                  | Definition      |
+| --------------------- | ----------------------------- |
+| mTLS | Mutual Transport Layer Security (mTLS) allows two parties to authenticate each other during the initial connection of an SSL/TLS handshake. |
+| UCPD | Universal Control Plane Daemon for Radius |
```

Comment:
Do we need to talk about `UCPD` specifically? It feels more accessible to talk about UCP as a system, than `UCPD` (UCP as a process/container).